servicecontrol/installation.md

```diff
@@ -79,6 +79,20 @@ Clicking the upgrade link will:
 * Run the new binaries to create required queues
 * Start the service
 
+
+To upgrade multiple majors the following approach needs to be taken:
+
+- Upgrade the the latest minor.patch release within a give major
```

Comment:
```suggestion
- Upgrade to the latest minor.patch release within a given major
```

Reply:
I would maybe even make this the current major

```suggestion
- Upgrade to the latest minor.patch release within a currently installed major
```